client/src/components/judging/JudgingRoom.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import roomData = require('../../data.json')
+
+
+interface JudgingRoomProps {
+    table: string
+}
+
+class JudgingRoom extends React.Component<JudgingRoomProps, {}>{
```

Comment:
Since this class has no state, in theory it should be stateless, but not a big deal